tests/Application/Actions/Hooks/StripeUpdateTest.php

```diff
@@ -24,16 +25,10 @@ public function testUnsupportedAction(): void
         $container = $app->getContainer();
 
         $body = file_get_contents(dirname(__DIR__, 3) . '/TestData/StripeWebhook/pi_canceled.json');
-        $donation = $this->getTestDonation();
         $webhookSecret = $container->get('settings')['stripe']['webhookSecret'];
         $time = (string) time();
 
         $donationRepoProphecy = $this->prophesize(DonationRepository::class);
-        $donationRepoProphecy
```

Comment:
We can probably remove this prophecy completely if it has no mocked methods left?

Reply:
@NoelLH When I remove this prophecy I get the following error:

`Error: Call to a member function setCampaignRepository() on null`

Reply:
Ah, OK @tbg-jasonfung. If you look at the trace of that call, it's to do with the 'real' `DonationRepository` setup's reliance on `EntityManagerInterface` in `repositories.php`. It looks like we do need to mock the donation repo in this case but not the EM, so we can remove that and it's DI override instead.